tests/fast-integration/eventing-test/eventing-test.js

```diff
@@ -123,6 +130,45 @@ describe('Eventing tests', function() {
     });
   }
 
+  function testStreamNotReCreated() {
+    context('stream check with JetStream backend', function() {
+      let wantStreamData = null;
+      let gotStreamData = null;
+      before('check if stream creation timestamp is available', async function() {
+        try {
+          const cm = await getConfigMap(streamDataConfigMapName);
+          wantStreamData = cm.data;
+        } catch (err) {
+          console.log('Skipping the stream recreation check due to missing configmap!');
+          this.skip();
+        }
+        if (skipStreamReCreationTest(wantStreamData)) {
+          console.log('Skipping the stream recreation check due to missing info!');
```

Comment:
Can you please add what info, the creation timestamp, is missing? Think of the poor soul who has to debug with this message.